django_u2f/views.py

```diff
@@ -12,7 +12,7 @@
 from django.contrib import auth, messages
 from django.conf import settings
 from django.http import HttpResponseRedirect
-from django.core.urlresolvers import reverse, reverse_lazy
+from django.urls import reverse, reverse_lazy
```

Comment:
What's the minimum version of django this requires?

Reply:
I believe it has been added in 1.10